ProjFS.Mac/PrjFSKext/public/PrjFSCommon.h

```diff
@@ -34,4 +34,14 @@ enum PrjFSServiceUserClientType
     UserClientType_Log,
 };
 
+// When building the kext in user space for unit testing, we want some functions
+// to have external linkage in order to make them testable
+#ifdef KEXT_UNIT_TESTING
+#define KEXT_TESTABLE_STATIC
+#define KEXT_TESTABLE_STATIC_INLINE
+#else
+#define KEXT_TESTABLE_STATIC static
+#define KEXT_TESTABLE_STATIC_INLINE static inline
```

Comment:
What about just `KEXT_STATIC` and `KEXT_STATIC_INLINE`. I'm not sure the `TESTABLE` thoughout the codebase adds much extra value. It's the comment on the defines that tells me what we're making this a macro.

Reply:
Yes!  @wilbaker and I actually talked about the same name change offline.  I'll make it happen.